packages/launchpad/src/components/Switch.vue

```diff
@@ -5,33 +5,23 @@
     @click="$emit('update', !value)"
   >
     <span
-      class="
-        absolute
-        block
-        toggle
-        border border-1 border-gray-300
-        rounded-md
-        bg-white
-      "
```

Comment:
Why making it flat? is it your editor? or your preference?

Reply:
Probably. We really need linting.